src/lib/nodes/abstract.node.ts

```diff
@@ -164,6 +197,7 @@ export abstract class Node<C = unknown> {
 
   getStatus() {
     return {
+      alt_ip: this.altIp,
```

Comment:
```suggestion
      altIp: this.altIp,
```

Use camelCase as for other properties.

Reply:
https://github.com/Adamant-im/adamant-im/pull/768/commits/259d74cfa6be278d00957dc111f98c64c3a070c9